applications/StructuralMechanicsApplication/tests/test_axis_projection.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import math
+
+class TestAxisProjection(KratosUnittest.TestCase):
+
+    def _set_up_test_case(self):
+        current_model = KratosMultiphysics.Model()
```

Comment:
this might be a bad mem-error!
this is a local-var that can go out of scope, but the returned ModelPart is still used!

I had such a nasty error during my master thesis, kept @roigcarlo and @jcotela busy for some time :D
Good times :DD

Reply:
so ... should I create the model in each test?